appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -17,7 +17,9 @@ type ExtendedSkuDescription = WebSiteManagementModels.SkuDescription & { label?:
 export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
         let skus: ExtendedSkuDescription[] = wizardContext.advancedCreation ? this.getCommonSkus() : this.getRecommendedSkus();
-        if (wizardContext.newSiteKind === AppKind.functionapp) {
+        if (wizardContext.newSiteKind.includes(AppKind.workflowapp)) {
```

Comment:
Where is `AppKind.workflowapp` set?

Have there been discussions on whether or not workflow/logic apps will be available in basic creation?  This will overwrite any skus we have from the step above.

Reply:
Today workflow app (in our GA) will only support Workflow Standard Skus and nothing else.
In case of App Service Plan it will only support ASE v3 plans, for which i am providing planfilter in the context.

Reply:
Could you move this to be the bottom of the if/else if conditions?  I don't think it should effect us, but I want to be safe and make sure that our check for `functionapp` happens before the `workflowapp`.

Reply:
though it doesnt matter, but i will move it.